packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -15,12 +15,18 @@ import useSetting from '../../components/use-setting';
 /**
  * Control for letter-spacing.
  *
- * @param {Object}   props          Component props.
- * @param {string}   props.value    Currently selected letter-spacing.
- * @param {Function} props.onChange Handles change in letter-spacing selection.
- * @return {WPElement}                      Letter-spacing control.
+ * @param {Object}   props                      Component props.
+ * @param {string}   props.value                Currently selected letter-spacing.
+ * @param {Function} props.onChange             Handles change in letter-spacing selection.
+ * @param {boolean}  props.__unstableInputWidth Input width to pass through to inner UnitControl.
+ *
+ * @return {WPElement}                          Letter-spacing control.
  */
-export default function LetterSpacingControl( { value, onChange } ) {
+export default function LetterSpacingControl( {
+	value,
+	onChange,
+	__unstableInputWidth = '60px',
```

Comment:
The JSDocs mark `__unstableInputWidth` as of type `boolean`, but the default value is `60px` — is this on purpose?